src/sentry/testutils/cases.py

```diff
@@ -1521,6 +1521,9 @@ def create_user_member_role(self):
 class TestMigrations(TransactionTestCase):
     """
     From https://www.caktusgroup.com/blog/2016/02/02/writing-unit-tests-django-migrations/
+
+    Note that when running these tests locally you will need to set the `MIGRATIONS_TEST_MIGRATE=1`
+    environmental variable for these to pass.
```

Comment:
Wording on 'pass'. Is it pass, or execute/run?

Reply:
They will run without `MIGRATIONS_TEST_MIGRATE=1` but will (likely) fail

Reply:
Maybe wording should be '.. environment variable for these to correct test migrations.'?